youtube_dl/extractor/zdf.py

```diff
@@ -70,6 +70,23 @@ def xml_to_format(fnode):
             '_available': is_available,
         }
 
+    def xml_to_thumbnails(fnode):
+        thumbnails = list()
+        for node in fnode:
+            width_x_height = node.attrib['key']
```

Comment:
Extraction should not fail due to changes in XML layout. If `key` is removed the whole extraction will be broken.